### ISSUE TITLE
API: return service creation parameters in GET service json

### ISSUE THREAD

**Issue by alx** [jolibrain/deepdetect#446]
After creating a new service, it should be possible to get service creation parameters when requesting information about a service.

# Example:

## Service creation

```
curl -X PUT "http://localhost:8080/services/myserv" -d "{\"mllib\":\"caffe\",\"description\":\"example classification service\",\"type\":\"supervised\",\"parameters\":{\"input\":{\"connector\":\"csv\"},\"mllib\":{\"template\":\"mlp\",\"nclasses\":9,\"layers\":[512,512,512],\"activation\":\"prelu\"}},\"model\":{\"repository\":\"/home/me/models/example\"}}"
```

## GET service information - current result

```
curl -X GET "http://localhost:8080/services/myserv"
{
  "status":{
         "code":200,
         "msg":"OK"
      },
  "body":{
         "mllib":"caffe",
         "description":"example classification service",
         "name":"myserv",
         "jobs":
        {
          "job":1,
          "status":"running"
        }
     }
}
```

## GET service information - expected result

```
curl -X GET "http://localhost:8080/services/myserv"
{
  "status":{
         "code":200,
         "msg":"OK"
      },
  "body":{
         "mllib":"caffe",
         "description":"example classification service",
         "name":"myserv",
         "jobs":
        {
          "job":1,
          "status":"running"
        },
       "parameters":
      {
        "input":{
          "connector":"csv"
        },
        "mllib":{
          "template":"mlp",
          "nclasses":9,
          "layers":[512,512,512],
          "activation":"prelu"
         }
       }
     }
}
```

**Comment by beniz** [comment]
Implemented in PR #451

**Comment by beniz** [comment]
merged.
